tsfile/src/main/java/org/apache/iotdb/tsfile/utils/Murmur128Hash.java

```diff
@@ -0,0 +1,153 @@
+package org.apache.iotdb.tsfile.utils;
```

Comment:
Add the license, please.

Reply:
done

Reply:
hi, I just want to double check that is this class implemented by you rathar than other projects..

Reply:
My question? @SilverNarcissus

Reply:
I will update license file.